pokemon-showdown

```diff
@@ -2,14 +2,16 @@
 'use strict';
 
 const child_process = require('child_process');
+const fs = require('fs');
+const path = require('path');
 
 // Make sure we're Node 6+
 
 try {
 	eval('{ let [a] = [1]; }');
 } catch (e) {
 	console.log("We require Node.js v6 or later; you're using " + process.version);
-	process.exit();
+	process.exit(0);
```

Comment:
...didn't notice that. I'll take a look at how Node deals with exit codes internally, since I'm not positive exiting with errno `EX_UNAVAILABLE` will do what we're intending to do, judging from the other exit codes below 128